kratos/utilities/math_utils.h

```diff
@@ -1876,6 +1876,56 @@ class MathUtils
         return is_converged;
     }
 
+    /**
+     * @brief Calculates the Factorial of a number k, Factorial = k!
+     * @tparam Number The number of which the Factorial is computed
+     */
+    template<class TIntegerType>
+    static inline TIntegerType Factorial(
+        const TIntegerType Number
+        )
```

Comment:
```suggestion
    static inline TIntegerType Factorial(const TIntegerType Number)
```
Style guide for one argument functions.